### PR TITLE
Fix code snippets

### PR DESCRIPTION
**Pull request by Youssef1313**
- Added language identifier to code snippets that doesn't have.
- A snippet that is written in vb and c# was explicitly saying that the following code is in visual basic, which I think was written like that before the switch between languages was there.
- Fixed some snippets had lines outside the code snippet (after the "```")
- Some changes to the code snippets to follow conventions.

### CODE REVIEW

**Comment by Youssef1313** [comment]
Thanks @mairaw , I've applied your changes. I also noticed some additional missing fixes and applied them.

**Comment by Youssef1313** [comment]
Applied your suggestions @mairaw, also noticed two small snippets without language identifier, added csharp identifier to them, and added equivalent vb sample, also replaced `If` with `if` in a C# code.

**Comment by Youssef1313** [comment]
@mairaw, Found another missing fix and applied it.


There is also another problem that I can't determine its reason.

Here is the content of the xml file.
image


Here is how it's rendered in the live docs.
image

**Comment by mairaw** [comment]
Oh yes! That sentence should be moved to under the ## Examples header.

**Comment by Youssef1313** [comment]
> 
> 
> Oh yes! That sentence should be moved to under the ## Examples header.

Thanks for confirmation! I've moved it under the examples header

File: xml/System.Data.Linq/DataContext.xml [hunk 1]
```diff
@@ -876,29 +883,29 @@ db.ExecuteCommand("UPDATE Products SET QuantityPerUnit = {0} WHERE ProductID = {
...
-Dim customers = db.ExecuteQuery(Of Customer)("SELECT CustomerID, _    CompanyName, ContactName, ContactTitle, _  
-   Address, City, Region, PostalCode, Country, Phone, Fax _  
-   FROM dbo.Customers _  
-WHERE City = {0}", "London")  
-  
-For Each c As Customer In customers  
-    Console.WriteLine(c.ContactName)  
-Next  
+```vb
+Dim customers = db.ExecuteQuery(Of Customer)("SELECT CustomerID, _    CompanyName, ContactName, ContactTitle, _
```
Comment (mairaw) [inline xml/System.Data.Linq/DataContext.xml]: One more thing I noticed if you don't mind 

```suggestion
Dim customers = db.ExecuteQuery(Of Customer)("SELECT CustomerID, _    
    CompanyName, ContactName, ContactTitle, _
```

File: xml/System.Data.OracleClient/OracleLob.xml [hunk 2]
```diff
@@ -53,15 +53,23 @@
...
  You can construct an <xref:System.Data.OracleClient.OracleLob> that is NULL using this format:  
   
-```  
+```csharp
 OracleLob myLob = OracleLob.Null;  
-```  
+```
+
+```vb
+Dim myLob As OracleLob = OracleLob.Null;
```
Comment (mairaw) [inline xml/System.Data.OracleClient/OracleLob.xml]:
```suggestion
Dim myLob As OracleLob = OracleLob.Null
```

File: xml/System.Data.OracleClient/OracleLob.xml [hunk 3]
```diff
@@ -53,15 +53,23 @@
...
   
-```  
-If(myLob == OracleLob.Null)  
-```  
+```csharp
+if (myLob == OracleLob.Null)  
+```
+
+```vb
+If (myLob = OracleLob.Null)
```
Comment (mairaw) [inline xml/System.Data.OracleClient/OracleLob.xml]:
```suggestion
If (myLob = OracleLob.Null) Then
```

**Comment by Youssef1313** [comment]
@mairaw, This is ready for final review.

**Comment by mairaw** [comment]
Kicking off a new build to be able to see build reports.

**Comment by Youssef1313** [comment]
@mairaw, Is this ready ?

**Comment by Youssef1313** [comment]
Ping @mairaw.

**Comment by mairaw** [comment]
Build reports expired. Building again.

**Comment by Youssef1313** [comment]
@mairaw, is there something left for this ?

**Comment by mairaw** [comment]
Let me review again. Sorry for the delay. Lost track of this one.

**Pull request merged**